app/models/perm.rb

```diff
@@ -12,6 +12,20 @@
 
 class Perm < ApplicationRecord
 
+  # =============
+  # = Constants =
+  # =============
+
+  ADD_ORGS            = Perm.where( name: "add_organisations" ).first.freeze
+  CHANGE_AFFILIATION  = Perm.where( name: "change_org_affiliation" ).first.freeze
+  GRANT_PERMISSIONS   = Perm.where( name: "grant_permissions" ).first.freeze
+  MODIFY_TEMPLATES    = Perm.where( name: "modify_templates" ).first.freeze
+  MODIFY_GUIDANCE     = Perm.where( name: "modify_guidance" ).first.freeze
+  USE_API             = Perm.where( name: "use_api" ).first.freeze
+  CHANGE_ORG_DETAILS  = Perm.where( name: "change_org_details" ).first.freeze
+  GRANT_API           = Perm.where( name: "grant_api_to_orgs" ).first.freeze
+  REVIEW_PLANS        = Perm.where( name: "review_org_plans" ).first.freeze
```

Comment:
You could do this in one query with something like:
```ruby
  # Preload all of the records and assign them to a Constant
  Perm.all.each { |perm| const_set(perm.name.upcase, perm) }
```
You would then have to alter a few of the shortcut methods below since names like `GRANT_API` would actually be `GRANT_API_TO_ORGS`